include/enums.h

```diff
@@ -6472,4 +6472,16 @@ enum Language {
     LANGUAGE_ES = 3,
 };
 
+enum IdleScriptState {
```

Comment:
Per the original issue, I
 think we wanted this one in script_api.h

Reply:
I've considered renaming them in the past, but decided against it because the vast, vast majority of calls do not use the RESET/RESTART options (3/117) for EnableActorBlur and (67/1060) for EnableIdleScript. I haven't found any alternative names to be a sufficiently compelling improvement over EnableX for a function which is nearly always used to enable and disable.